src/App.js

```diff
@@ -1,23 +1,50 @@
-import logo from './logo.svg';
 import './App.css';
+import { useState } from 'react';
 
+import NameInput from './Name/NameInput';
+import DrinkDropDown from './FoodOrder/DrinkDropDown';
+import EntreeDropDown from './FoodOrder/EntreeDropDown';
+import SideDropDown from './FoodOrder/SideDropdown';
+import Order from './FoodOrder/Order';
+import OrderInstructionsForm from './Instructions/OrderInstructionsForm';
+import OrderInstructionList from './Instructions/OrderInstructionsList';
 function App() {
+  const [orderName, setOrderName] = useState('');
+  const [drinkId, setDrinkId] = useState('1');
+  const [entreeId, setEntreeId] = useState('1');
+  const [sideId, setSideId] = useState('1');
+  const [instructions, setInstructions] = useState(['']);
```

Comment:
we don't need the empty string inside the array on line 16. Just `useState([])` works.